### PR TITLE
Add python client

### PR DESCRIPTION
**Pull request by shimk52**

### CODE REVIEW

File: Python/setup.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setuptools.setup(
+    name="timbermill",
+    version="0.0.1",
+    author="Shimon Klebanov",
+    author_email="shimonkle@gmail.com",
```
Comment (aperfectpoint) [inline Python/setup.py:13]: put your salesforce email

**Review by aperfectpoint — CHANGES_REQUESTED** [review]
@shimk52 I don't think we need to move every down to Java, let's just add a `python` directory (notice the lower-case).
@edengal - maybe we'll put all the clients under `timbermill-client`?

**Comment by shimk52** [comment]
@aperfectpoint I'm fine with both
The `Java` directory was per @edengal's request

**Comment by edengal** [comment]
@aperfectpoint the root folder is currently a maven root dir with a pom file, doesn't make sense to add another completely different project under the same path as the root to the maven project, same goes for the client, I rather we not mixed those.

**Comment by edengal** [comment]
@shimk52 Can you please change the dirs name to `timbermill-java` `timbermill-python`

**Comment by shimk52** [comment]
@edengal Done

**Review by aperfectpoint — APPROVED** [review]

**Review by edengal — APPROVED** [review]

**Pull request merged**